docs/en_us/internal/testing.rst

```diff
@@ -208,10 +208,15 @@ To run a single test format the command like this.
 
     paver test_system -t lms/djangoapps/courseware/tests/tests.py:ActivateLoginTest.test_activate_login
 
-The ``lms`` suite of tests runs concurrently, and with randomized order, by default.
-You can override these by using ``--no-randomize`` to disable randomization,
-and ``--processes=N`` to control how many tests will run concurrently (``0`` will
-disable concurrency). For example:
+The ``lms`` suite of tests runs with randomized order, by default.
+You can override these by using ``--no-randomize`` to disable randomization.
+
+You can also enable test concurrency with the ``--processes=N`` flag (where ``N``
```

Comment:
up to you, but we could mention here that the recommendation is to use the same # of processes as the # of cores.